weatherreport.cpp

```diff
@@ -1,4 +1,4 @@
-#include <assert.h>
+/*#include <assert.h>
```

Comment:
Hint: Make multiple SensorStub instances. Instead of returning fixed values, make a constructor to accept the values to return, and remember them in member variables. Return those variables when called.

Reply:
Thanks for the hint. We will work on it and submit soon.